server/localStorage.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from "./types";
```

Comment:
I'm not sure what localStorage is on the server-side, localStorage is a web client thing right? the server shouldn't know anything about this, nor should it have any effects here.
the server shouldn't persist the "current user's ID" anywhere, the database is its only state.